.circleci/config.yml

```diff
@@ -0,0 +1,72 @@
+version: 2.1
+
+executors:
+  default:
+    working_directory: ~/repo
+    description: The official CircleCI Ruby Docker image
+    docker:
+      - image: circleci/ruby:2.7.2
+        environment:
+          BUNDLER_VERSION: 2.2.26
+
+
+cashes:
+  - &bundle_cache_full v2-repo-{{ checksum "Gemfile.lock" }}
+  - &bundle_cache v2-repo-
```

Comment:
Do we need second cache?

Reply:
The first cache checks whether the Gemfile.lock has been changed and if not, then the second cache will not work and nothing will be downloaded
The second cache takes gems that were already installed, so it will download only gems we didn't have in the last commit.